<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trade</class>
 <widget class="QMainWindow" name="Trade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Vani&quot;;</string>
     </property>
     <property name="text">
      <string>player1</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Vani&quot;;
background-color : transparent;</string>
     </property>
     <property name="title">
      <string>give:</string>
     </property>
     <widget class="QLabel" name="label5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/brick2.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/grain.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/lumber.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/ors.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/wool.jpg</pixmap>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 213, 106);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="request">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>380</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Algerian&quot;;
color: rgb(209, 0, 0);
background-color: rgb(255, 205, 55);
border-color: rgb(255, 205, 55);
selection-background-color:rgb(255, 205, 55);</string>
     </property>
     <property name="text">
      <string>request</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../Favorites/Desktop/catan images/trade/bell5.png</normaloff>../../../../../../Favorites/Desktop/catan images/trade/bell5.png</iconset>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Vani&quot;;</string>
     </property>
     <property name="title">
      <string>players:</string>
     </property>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/player photo/girl1.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/player photo/boy2.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/player photo/boy1.jpg</pixmap>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Vani&quot;;</string>
      </property>
      <property name="text">
       <string>player2</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Vani&quot;;</string>
      </property>
      <property name="text">
       <string>player4</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Vani&quot;;</string>
      </property>
      <property name="text">
       <string>player3</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/bank4.jpg</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_65">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Vani</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 205, 55);
color: rgb(121, 0, 0);</string>
     </property>
     <property name="text">
      <string>choose cards and players who you want to trade with :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_66">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../Favorites/Desktop/catan images/player photo/bigGirl2.jpg</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Vani&quot;;</string>
     </property>
     <property name="title">
      <string>get:</string>
     </property>
     <widget class="QLabel" name="label13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/brick2.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/grain.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label15">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/lumber.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label16">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/ors.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label17">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../../Favorites/Desktop/catan images/resourceCards/cut/wool.jpg</pixmap>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 213, 106);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 222, 137);</string>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_72">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../Downloads/catan-logo.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Algerian&quot;;
color: rgb(209, 0, 0);
background-color: rgb(255, 205, 55);
border-color: rgb(255, 205, 55);
selection-background-color:rgb(255, 205, 55);</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../../Favorites/Desktop/catan images/trade/back1.png</normaloff>../../../../../../Favorites/Desktop/catan images/trade/back1.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
